source/write/update.txt

```diff
@@ -61,6 +61,19 @@ Each update method requires the following parameters:
   :manual:`Field Update Operators guide page</reference/operator/update-field/>` in the
   {+mdb-server+} manual.
 
+Upsert Option
```

Comment:
S: I would place this upsert section after the update one and update many example sections since this page is titled "Update Documents" and upsert is more of a cool extra function.